core/ledger/kvledger/channelinfo_provider.go

```diff
@@ -45,9 +45,7 @@ func (p *channelInfoProvider) NamespacesAndCollections(vdb statedb.VersionedDB)
 	for _, ccInfo := range chaincodesInfo {
 		ccName := ccInfo.Name
 		retNamespaces[ccName] = []string{}
-		for _, implicitCollName := range implicitCollNames {
-			retNamespaces[ccName] = append(retNamespaces[ccName], implicitCollName)
-		}
+		retNamespaces[ccName] = append(retNamespaces[ccName], implicitCollNames...)
```

Comment:
we can also make it `retNamespaces[ccName] = append([]string, implicitCollNames...)` and remove line 47.